File: MyHeart.Web/ClientApp/models/professionInfo/bonds/CommercialNamePharmacyDto.ts
import PharmacyDto from "../PharmacyDto";

export default class CommercialNamePharmacyDto {
    commercialNamesId: number;
    pharmacyId: number;
    bondStrength: number;
    pharmacy: PharmacyDto;

    constructor(commercialNamesId: number, bondStrength: number, pharmacy: PharmacyDto) {
        this.commercialNamesId = commercialNamesId;
        this.pharmacyId = pharmacy.id;
        this.pharmacy = pharmacy;
        this.bondStrength = bondStrength;
    }
    
}

File: MyHeart.DTO/ProfessionInformation/CommercialNamesBonds/CommercialName_PharmacyDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MyHeart.DTO.ProfessionalInformation
{
    public class CommercialName_PharmacyDTO : BaseEntityDTO
    {
        public int CommercialNamesId { get; set; }
        public int PharmacyId { get; set; }
        public int BondStrength { get; set; }
    }
}
